BrainPortal/app/views/data_providers/show.html.erb

```diff
@@ -70,24 +70,24 @@
       <% t.empty_cell %>
     <% end %>
 
-    <% t.edit_cell(:online, :content => (@provider.online ? "Online" : "Offline")) do %>
-      <%= select_tag "data_provider[online]", options_for_select([["Online", true], ["Offline", false]], :selected => @provider.online) %>
+    <% t.edit_cell(:online, :content => (@provider.online ? "Online" : "Offline")) do |f| %>
+      <%= f.select :online, [["Online", true], ["Offline", false]] %>
     <% end %>
 
     <% t.edit_cell(:user_id, :header => "Owner", :content => link_to_user_with_tooltip(@provider.user), :disabled => ! current_user.has_role?(:admin_user) ) do %>
       <%= user_select("data_provider[user_id]", { :selector => @provider } ) %>
     <% end %>
 
-    <% t.edit_cell(:online, :content => (@provider.read_only ? "Read Only" : "Read/Write"), :header => "Mode") do %>
-      <%= select_tag "data_provider[read_only]", options_for_select([["Read/Write", false], ["Read Only", true]], :selected => @provider.read_only) %>
+    <% t.edit_cell(:online, :content => (@provider.read_only ? "Read Only" : "Read/Write"), :header => "Mode") do |f| %>
```

Comment:
There is a bug here that was not introduced by this PR but should be fixed: the first arg of edit_cell has the wrong symbol (`:online` instead of `:read_only`)

This also happens at line 89, for the attribute `:not_syncable`